src/utils.ts

```diff
@@ -102,9 +102,20 @@ export const getFileIdRegexp = () => {
 
 export const FILE_ID_REGEXP = getFileIdRegexp();
 
-export const getFileTypesSetting = () => {
+export const getFileSearchExpression = (): string => {
+  const configFileTypes: string[] = getConfiguration("fileTypes");
```

Comment:
We should probably either deprecate this search option or include it to `package.json` and README.